docs/data/joy/components/radio/radio.md

```diff
@@ -95,40 +106,17 @@ Use the CSS variable `--Radio-action-radius` to control the border radius of the
 
 ### Icon
 
-`Radio`, by default, comes without an unchecked component.
+Radio, by default, comes without an unchecked component.
 To add an icon to both unchecked and checked states, use the `uncheckedIcon` and `checkedIcon` props.
```

Comment:
This section doesn't do a great job of explaining this feature IMO. Here's how I'd rewrite it.

```suggestion
Use the `checkedIcon` and/or `uncheckedIcon` props to add custom icons to the Radio button that correspond to each state, respectively.
The demo below shows how to apply a custom `checkedIcon` to the Radio button that sits on the corner of each Sheet:
```